### ISSUE TITLE
New Feature: Use Custom GPT with anki smart notes

### ISSUE THREAD

**Issue by hienstorfer** [piazzatron/anki-smart-notes#11]
There is a lot of custom GPTs out there with specific knowledge that could be used with anki-smart-notes.
Also, it is possible to create an own GPT using uploaded documents.
Capability to use these GPT would be really cool!
<hr>

<img width="625" alt="image" src="https://github.com/piazzatron/anki-smart-notes/assets/55869471/448ac061-eb30-4b81-93ab-cff4aa36428c">

**Comment by piazzatron** [comment]
Good idea. Would pair nicely with other features to specialize the GPT - i.e. an option to prefix all prompts with user defined text (similar to how the ChatGPT app does it, essentially making a custom GPT within SmartNotes), and sending up the full context of your note to ChatGPT. On the todo list!
